Sources/ProjectDescription/CopyFilesAction.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+public struct CopyFilesAction: Equatable, Codable {
+    public var name: String
```

Comment:
I don't think we want to let the users name the action, let's do that automatically

Reply:
I've added a comment with more context, but basically this is important so we can organize the phases and understand what is going on, this name is visible.